### PR TITLE
turn on app cache headers

### PR DESCRIPTION
**Pull request by camallen**
_Please request review from `@zooniverse/frontend` team. If PR is related to design, please request review from `@beckyrother` in addition._ 

Package:

Turn on the cache headers added in #950 and #1812, allow the app to specify how long resources should be cached in CDN, browsers for.

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?

## Components
- [ ] Has a storybook story been created or updated?
- [ ] Is the component accessible? 
  - [ ] Can it be used with a screen reader? [BBC guide to testing with VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
  - [ ] Can it be used from the keyboard? [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - [ ] Is it passing accessibility checks in the storybook?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `yarn panic && yarn bootstrap` or `docker-compose up --build` and app works as expected?

## Publishing

- [ ] Is the changelog updated?
- [ ] Are the dependencies updated for apps and libraries that are using the newly published library?

## Post-merging

- [ ] Did the app deploy to https://frontend.preview.zooniverse.org/projects/:project-name/:owner or https://frontend.preview.zooniverse.org/about?
- [ ] Is the new feature working or bug now fixed?
  - [ ] Is there a Talk or blog post written to announce the new feature(s)?
- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
  - [ ] Is this approved by our designer?
- [ ] Is this ready for production deployment?

### CODE REVIEW

File: kubernetes/deployment-staging.tmpl [hunk 1]
```diff
@@ -155,6 +157,8 @@ spec:
               value: staging
             - name: SENTRY_DSN
               value: https://1f0126a750244108be76957b989081e8@sentry.io/1492498
+            - name: ENABLE_CACHE_HEADERS
+              value: true
```
Comment (srallen) [inline kubernetes/deployment-staging.tmpl:161]: Non-blocking question just for my understanding: Does this get evaluated as a string? The conditional in the code checks for this value to be `'true'`

**Review by srallen — APPROVED** [review]
Non-blocking: could we get this documented in the env var section of the readme? https://github.com/zooniverse/front-end-monorepo#environment-variables

File: kubernetes/deployment-staging.tmpl (hunk 1 above)
Comment (camallen) [reply kubernetes/deployment-staging.tmpl:161]: my understanding is it's YAML so it does but i keep getting this wrong. See what the dry run does and then what the pod env vars are set to after deploy. Ensuring strings would be "true" i believe.

Comment (zwolf) [reply kubernetes/deployment-staging.tmpl:161]: Pretty sure numbers and booleans need to be cast to strings in templates.

Comment (camallen) [reply kubernetes/deployment-staging.tmpl:161]: OK - please add suggestions to fix it and merge.

File: kubernetes/deployment-staging.tmpl [hunk 2]
```diff
@@ -65,6 +65,8 @@ spec:
               value: staging
             - name: SENTRY_DSN
               value: https://2a50683835694829b4bc3cccc9adcc1b@sentry.io/1492691
+            - name: ENABLE_CACHE_HEADERS
+              value: true
```
Comment (srallen) [inline kubernetes/deployment-staging.tmpl:69]:
```suggestion
              value: 'true'
```

File: kubernetes/deployment-staging.tmpl (hunk 1 above)
Comment (srallen) [reply kubernetes/deployment-staging.tmpl:161]:
```suggestion
              value: 'true'
```

Comment (eatyourgreens) [reply kubernetes/deployment-staging.tmpl:161]: We've got bare numbers elsewhere in the template but I don't know. 🤷‍♂️

Comment (zwolf) [reply kubernetes/deployment-staging.tmpl:161]: I'm talking specifically about the setting of environment variables.

**Pull request merged**